attest/secureboot.go

```diff
@@ -144,7 +144,13 @@ func ParseSecurebootState(events []Event) (*SecurebootState, error) {
 		case internal.EFIVariableAuthority:
 			a, err := internal.ParseUEFIVariableAuthority(bytes.NewReader(e.Data))
 			if err != nil {
-				return nil, fmt.Errorf("failed parsing EFI variable authority at event %d: %v", e.sequence, err)
+				// Workaround for: https://github.com/google/go-attestation/issues/157
+				if err == internal.ErrSigMissingGUID {
+					// Trim off an erroneous additional byte, and re-do the digest check.
+					digestVerify = e.digestEquals(e.Data[:len(e.Data)-1])
```

Comment:
For this case, are you okay with a.Certs being nil? It looks like `ParseUEFIVariableAuthority` throws away the return value from `parseEfiSignature` if the later returns an error.

Reply:
Whelp, good catch! ^^' I'll fix + make sure the test checks the cert.